charts/web3signer/values.yaml

```diff
@@ -294,5 +294,5 @@ slashingprotectiondb:
           done
     persistence:
       # -- Uses an EmptyDir when not enabled
-      enabled: false
+      enabled: true
       size: 8Gi
```

Comment:
8Gi feels like way too much for this usecase?

Reply:
Yeah, I currently have no clue how big this DB grows. It should be way less than 8Gi. Probably 1Gi is more than enough. I wouldn't go less than 1Gi just because most cloud providers won't allow you to allocate less than that. 

Once we start running this on our test clusters for a while we'll get a better feeling on how much data is needed. I would expect this to be just some MBs.